johnsonandjohnson/Bodiless-JS
# Fix missing top menu if not on home docs page issue.

## Changes

Add alias configure for _navbar.md

## Test Instructions

- Open docs from edit site (upper left menu).
- Navigate to pages.
- Verify nav menu stay on top across different pages.

## Related Issues

Fixes: #173
